manager/integration/tests/test_infra.py

```diff
@@ -239,9 +239,22 @@ def test_offline_node(reset_cluster_ready_status):
     print(f'==> stop node: {node_name}')
 
     cloudprovider.instance_stop(node)
-    wait_for_node_down_aws(cloudprovider, node)
-    k8s_node_down = wait_for_node_down_k8s(node_name, k8s_api_client)
+    aws_node_down = wait_for_node_down_aws(cloudprovider, node)
+    if aws_node_down:
+        print(f'AWS node is down in {RETRY_COUNTS} sec')
+    else:
+        print(f'AWS node not down in {RETRY_COUNTS} sec')
+        print(f'Check AWS node status again in {RETRY_COUNTS} sec')
+        aws_node_down = wait_for_node_down_aws(cloudprovider, node)
```

Comment:
Seems the instance just needs more time to stop? Why not just change `RETRY_INTERVAL` to `RETRY_INTERVAL_LONG`?

Reply:
I was set to print some information on console output, I didn’t realize that we have `RETRY_INTERVAL_LONG` in the common.py. I would modify my PR.